docs/data/material/varnish/header/header.md

```diff
@@ -1,39 +1,38 @@
 ---
-title: Varnish Header
+title: AI2 Varnish Header
 githubLabel: 'varnish: header'
 ---
 
 # Header
 
 <p class="description">Official AI2 Header.</p>
 
-
 ## When To Use
 
 You should put this at the top of the page. It is highly recommended to be present on every page of
 your application.
 
 ### API
 
-| Property      | Description                                                         | Type    | Default |
-| ------------- | ------------------------------------------------------------------- | ------- | ------- |
-| bannerAlwaysVisible | Prevents the AI2 banner from disappearing when the page is scrolled | boolean | false   |
-| customBanner | A Custom AI2 banner that you can inject into the header via this prop | JSX / TSX Element; if left undefined, the banner is the default AI2 banner | undefined |
-| scrollTriggerTarget | If you choose to use the AI2 Header with Smart Banner within a div, you can change the scroll target to be the specific div it is in. Note: This should rarely be used, reach out to the team if you find yourself using this often. | HTMLDivElement | undefined |
+| Property            | Type                                                                       | Default   | Description                                                                                                                                                                                                                          |
```

Comment:
@arnavic to match material, we put description on hte end (different than the current varnish docs)

Reply:
noted! thanks